### DISCUSSION TITLE
League mode rotation

### DISCUSSION THREAD

**Discussion by Osrepnay** [BattlesnakeOfficial/feedback#102] [General]
I think it would be a good idea to rotate the league's gamemodes occasionally, that way each season would be different from the last one and the snakes would have to adapt each league (unless it's already a god-amongst-snakes snake)

**Comment by bvanvugt** [comment]
Yeah! We'd like something like this too, while also respecting that some players don't want to develop for a new game mode every league (this happened in Spring League, when we ran a parallel royale ladder).

One idea kicking around is to keep Royale as our primary competitive mode, and introduce experimental ladders each league.

We're super open to ideas for supporting and promoting new game modes regularly.
